examples/CMakeLists.txt

```diff
@@ -22,13 +22,14 @@ target_link_libraries(${PROJECT_NAME} PRIVATE cappuccino ${CAPPUCCINO_SYSTEM_LIB
 target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
 
 ### lfuda_simple ###
-project(lfuda_simple CXX)
-set(LFUDA_SIMPLE_SOURCE_FILES
-    examples/lfuda_simple.cpp
-)
-add_executable(${PROJECT_NAME} ${LFUDA_SIMPLE_SOURCE_FILES})
-target_link_libraries(${PROJECT_NAME} PRIVATE cappuccino ${CAPPUCCINO_SYSTEM_LIBRARY_DEPENDENCIES})
-target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
+# TODO disabled due to compiler errors
+#project(lfuda_simple CXX)
+#set(LFUDA_SIMPLE_SOURCE_FILES
+#    examples/lfuda_simple.cpp
+#)
+#add_executable(${PROJECT_NAME} ${LFUDA_SIMPLE_SOURCE_FILES})
+#target_link_libraries(${PROJECT_NAME} PRIVATE cappuccino ${CAPPUCCINO_SYSTEM_LIBRARY_DEPENDENCIES})
+#target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
```

Comment:
Should we remove this or can it be uncommented?

Reply:
I left it commented as a reminder so it can be added back once the errors are fixed -- fixing them here seems out of scope to me

Reply:
Got it, that makes sense